qiita_db/analysis.py

```diff
@@ -266,6 +275,21 @@ def name(self):
         return conn_handler.execute_fetchone(sql, (self._id, ))[0]
 
     @property
+    def portal(self):
```

Comment:
Can be useful for debugging, and is a small thing that is easy to maintain. I'd rather have access to it through the object if for nothing else than an API measure.